Jenkinsfile

```diff
@@ -37,16 +43,28 @@ timestamps {
 		  }
 
 		  stage('Deploy') {
-				sh "./ci/deploy.sh --branch=${encodedBranchName}"
+
+              ansiColor('xterm') {
+
+                  sh "./ci/deploy.sh --branch=${encodedBranchName}"
+
+              }
+
 		  }
 
 		  stage('Run e2e tests') {
-				try {
-					 sh "./ci/runE2ETests.sh --branch=${encodedBranchName}"
-				} finally {
-					 sh "./ci/deploySelfDocu.sh --branch=${encodedBranchName}"
-					 junit 'scenarioo-client/test-reports/*.xml'
-				}
+
+		      ansiColor('xterm') {
+
+                try {
+                     sh "./ci/runE2ETests.sh --branch=${encodedBranchName}"
+                } finally {
+                     sh "./ci/deploySelfDocu.sh --branch=${encodedBranchName}"
+                     junit 'scenarioo-client/test-reports/*.xml'
+                }
+
+			  }
+
```

Comment:
Sorry for this strange indentation - in my intellij it looks ok, but intellij tooling for jenkinsfile maybe missing?
We really have to do something about our intellij settings.
maybe it is just the web view that displays tba in strange way.